channels/smartcard/client/smartcard_pack.c

```diff
@@ -3796,7 +3796,7 @@ LONG smartcard_pack_locate_cards_return(SMARTCARD_DEVICE* smartcard, wStream* s,
 {
 	LONG status;
 	DWORD cbDataLen = ret->cReaders;
-	DWORD index = 0;
+	UINT32 index = 0;
```

Comment:
This should not be necessary, `UINT32` and `DWORD` are both defined to the same 32bit unsigned integer type, if not then something in winpr is missing for MinGW